pdns/lua-record.cc

```diff
@@ -966,21 +966,22 @@ static void setupLuaRecords(LuaContext& lua) // NOLINT(readability-function-cogn
   lua.writeFunction("createForward6", []() {
       DNSName rel=s_lua_record_ctx->qname.makeRelative(s_lua_record_ctx->zone);
       auto parts = rel.getRawLabels();
+      string address = "::";
+
       if(parts.size()==8) {
         string tot;
```

Comment:
In order to make the code simpler to read, it would be worth getting rid of that `tot` variable and operate on `address` directly, after clearing it (`address.clear()`).